### PR TITLE
Set max width for dashboard content

### PR DESCRIPTION
**Pull request by meowwwls**
Otherwise on very large screens, the chart/chart placeholder just stretches endlessly.

### CODE REVIEW

File: pages/dashboard.js [hunk 1]
```diff
@@ -280,10 +281,11 @@ const Dashboard = () => {
             borderRadius='0'
             color='ocean'
             fontSize='1.5rem'
-            onClick={() => downloadData(
-              JSON.stringify(userInstallData),
-              'flossbank_user_data.json'
-            )}
+            onClick={() =>
```
Comment (joelwass) [inline pages/dashboard.js:284]: idk how your commits pass but this breaks the pre commit hook of standard! not sure why it keeps getting changed :( i can't even commit when it's like this

File: pages/dashboard.js [hunk 2]
```diff
@@ -173,9 +173,10 @@ const Dashboard = () => {
       <Section
         backgroundColor='lightRock'
         display={{ md: 'grid' }}
-        gridTemplateColumns={{ lg: 'minmax(16rem, 20rem) 1fr' }}
+        gridTemplateColumns={{ lg: 'minmax(16rem, 20rem) minmax(auto, 70rem)' }}
```
Comment (joelwass) [inline pages/dashboard.js:176]: why is the first column 'minmax(16rem, 20rem)' ? isn't that pretty large for the data cards?

Comment (meowwwls) [reply pages/dashboard.js:176]: It's the same size as you see in production now. All I'm changing here is setting a max width for the chart section.

File: pages/dashboard.js (hunk 1 above)
Comment (meowwwls) [reply pages/dashboard.js:284]: I'm not sure what's going on here TBH. :\

Comment (joelwass) [reply pages/dashboard.js:284]: what happens when you run `npx standard` with this code, does it yell at you that indentation is wrong?

Comment (meowwwls) [reply pages/dashboard.js:284]: Had to temporarily disable format on save in VS code until I can find a better solution. I'll push the changes in a minute.

File: pages/dashboard.js (hunk 2 above)
Comment (joelwass) [reply pages/dashboard.js:176]: yea i get that, but i dont understand how min width for the cards is 16rem. i see that it somehow looks ok, i just don't understand _how_ it's working lol

File: pages/dashboard.js (hunk 1 above)
Comment (meowwwls) [reply pages/dashboard.js:284]: > what happens when you run `npx standard` with this code, does it yell at you that indentation is wrong?

I already changed it so I can't run and check, but I haven't been getting warnings about indentation when committing.

File: pages/dashboard.js (hunk 2 above)
Comment (meowwwls) [reply pages/dashboard.js:176]: 16rem is 256px and the smallest mobile screen is ~ 320px, so 16rem is less than full width on mobile.

Comment (joelwass) [reply pages/dashboard.js:176]: oh wow i didn't realize that. i guess i was thinking 16rem was way bigger! nvm

**Review by joelwass — APPROVED** [review]

**Pull request merged**